packages/flutter_tools/test/commands.shard/hermetic/run_test.dart

```diff
@@ -419,7 +419,7 @@ void main() {
         expect(
           testLogger.statusText,
           containsIgnoringWhitespace(
-            userMessages.flutterMissPlatformProjects(
+            UserMessages().flutterMissPlatformProjects(
```

Comment:
if you want to avoid a global, I think the correct thing to do here is to construct this before running the test, and ensure you're using the exact same instance in the expectation that the code under test used.

Reply:
or the easy way to maintain semantic equivalence would be to make this `globals.userMessages`

Reply:
I did the latter